addons/mail/models/mail_channel.py

```diff
@@ -335,10 +336,10 @@ def add_members(self, partner_ids=None, guest_ids=None, invite_to_rtc_call=False
                 user = channel_partner.partner_id.user_ids[0] if channel_partner.partner_id.user_ids else self.env['res.users']
                 # notify invited members through the bus
                 if user:
-                    self.env['bus.bus']._sendone(channel_partner.partner_id, 'mail.channel/joined', {
+                    notifications.append([channel_partner.partner_id, 'mail.channel/joined', {
```

Comment:
each item pushed should be a tuple rather than an array (both work but tuple is "more correct" in this case)

Reply:
Done !